Clarify compatibility flags and DNS documentation (#22091)
cloudflare/cloudflare-docs: src/content/compatibility-flags/nodejs-als.md

disable_flag: "no_nodejs_als"
---

> The Node.js [AsyncLocalStorage](https://nodejs.org/api/async_hooks.html#async_hooks_class_asynclocalstorage) API is enabled in Workers, allowing you to store data accessible to all asynchronous operations within a given execution context. This is useful for storing data that is relevant to the current request, such as request-specific metadata or tracing information.

Reviewer: On this one I think the original wording of the first sentence makes more sense. Use of this flag enables the API... it's not always enabled as the new wording may suggest.

> The Node.js [AsyncLocalStorage](https://nodejs.org/api/async_hooks.html#async_hooks_class_asynclocalstorage) API is enabled in Workers, allowing you to store data accessible to all asynchronous operations within a given execution context. This is useful for storing data that is relevant to the current request, such as request-specific metadata or tracing information.

Reviewer 2: ```suggestion
Enables the availability of the Node.js [AsyncLocalStorage](https://nodejs.org/api/async_hooks.html#async_hooks_class_asynclocalstorage) API in Workers.
```
